extensions/wikia/Recirculation/templates/Recirculation_container.php

```diff
@@ -0,0 +1 @@
+<section id="<?= $containerId ?>"></section>
```

Comment:
This variable will have to be escaped, otherwise it will be vulnerable to reflected XSS via `wikia.php`.